doonut-app-external-api/src/main/kotlin/com/doonutmate/example/controller/dto/ExampleCreateReq.kt

```diff
@@ -1,5 +1,9 @@
 package com.doonutmate.example.controller.dto
 
-data class ExampleCreateReq (
-    val name: String
-)
\ No newline at end of file
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "example 생성용 DTO")
+data class ExampleCreateReq(
+    @Schema(description = "이름")
```

Comment:
여기에서도 title과 description 중 어떤 걸 사용하는게 더 좋을까요?

Reply:
저는 개인적으로 swagger를 사용할 때 field에 붙이는 `@Schema` annotation에 example도 작성하는 편이에요!
사실 description을 정말 자세하게 작성해주면 좋긴하지만 보통은 field를 한글 단어로 직역하는 것이 대부분인 것 같아서 테스트할 때 example 같은 예시 데이터가 있으면 좋을 것 같아요!

Reply:
저도 example을 붙이는 걸 선호합니다! 그럼 여기도 example 붙이는 걸로 가시죠!  
```suggestion
    @Schema(description = "이름", example = "jichan")
```

Reply:
좋습니다!!